libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/GetActivityMembers.cs

```diff
@@ -113,5 +110,31 @@ protected override string OnComputeId()
         {
             return $"{GetType().Name}[{this.ActivityId?.ToString() ?? string.Empty},{this.Property?.ToString() ?? string.Empty}]";
         }
+
+        private async Task<IList<ChannelAccount>> GetActivityMembersAsync(ITurnContext turnContext, string activityId, CancellationToken cancellationToken)
+        {
+            // If no activity was passed in, use the current activity.
+            if (activityId == null)
+            {
+                activityId = turnContext.Activity.Id;
+            }
+
+            if (turnContext.Activity.Conversation == null)
+            {
+                throw new ArgumentException($"{nameof(GetActivityMembers)}.{nameof(GetActivityMembersAsync)}(): missing conversation");
+            }
+
+            if (string.IsNullOrWhiteSpace(turnContext.Activity.Conversation.Id))
+            {
+                throw new ArgumentException($"{nameof(GetActivityMembers)}.{nameof(GetActivityMembersAsync)}(): missing conversation.id");
+            }
+
+            var connectorClient = turnContext.TurnState.Get<IConnectorClient>();
```

Comment:
Is the ConnectorClient in TurnState when the adapter is BotFrameworokHttpAdapter?